depends/patches/native_qt/dont_hardcode_pwd.patch

```diff
@@ -0,0 +1,51 @@
+Do not assume FHS in scripts
```

Comment:
Given that this is introducing a second copy of a patch we already have, it'd be good if there was someone we could avoid having to maintain two different copies of the same thing.

Reply:
Fixed in https://github.com/bitcoin/bitcoin/pull/30997#issuecomment-2402990346.